c7n/resources/appflow.py

```diff
@@ -46,7 +46,7 @@ class TagAppFlowResource(Tag):
     permissions = ('appflow:TagResource',)
 
     def process_resource_set(self, client, resources, new_tags):
-        tags = {t.get('Key'): t.get('Value') for t in new_tags}
+        tags = convert_tags(new_tags, dict)
```

Comment:
On a couple of these, it's not clear if the original reference was to a list or a dict. Like for example the way this reads I have the impression that this was iterating a List<Tag> here so should it be `convert_tags(new_tags, dict)` instead? Or did you dig in a bit deeper to confirm already??

Reply:
the argument to `convert_tags` is the desired form of the output.  the whole point of the function is that it will automatically convert no matter which form the input is in.

Reply:
Oh wow, totally misread this originally. I see that now. Thanks for confirming!